src/hooks/HomeRefresh.ts

```diff
@@ -1,5 +1,5 @@
+import { DeviceEventEmitter } from 'expo-modules-core';
```

Comment:
why is it necessary to get that module from expo now instead of react-native like before? who or what was changing for this?

Reply:
https://blog.expo.dev/whats-new-in-expo-modules-infrastructure-7a7cdda81ebc

`react-native-unimodules` is deprecated (and removed) now, and the previous import took it from there.

Reply:
got it